pages/api/trigger-reminder.ts

```diff
@@ -0,0 +1,82 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { CloudTasksClient, protos } from '@google-cloud/tasks'
+import { auth } from 'google-auth-library'
+
+const project = process.env.GCP_PROJECT as string
+const queue = process.env.GCP_TASK_QUEUE as string
+const location = process.env.GCP_PRIMARY_REGION as string
+const cloudFunctionUrl = process.env.GCP_CLOUD_FUNCTION_URL_SEND_REMINDER as string
+const cloudFunctionEmail = process.env.GCP_CLOUD_FUNCTION_SERVICE_ACCOUNT_EMAIL as string
+const inSeconds = 0
+
+/**
+ * Normally, one might read credentials from a JSON file, or simply use Google Cloud functionality for auth.
+ * In our case, to run this locally and on Vercel, we pass in the credentials as an environment variable.
+ * Base64-encoding is used to avoid any issues due to newlines and so on.
+ * This is NOT encrypted, and by default does not expire, so handle with care (we can always disable keys if needed).
+ * The value was generated with "cat credentials.json | base64 -w 0 > base64_credentials.txt"
+ */
+const gcpCredentialBase64 = process.env.GCP_SERVICE_ACCOUNT_CREDENTIALS_BASE64 as string
+const credential = JSON.parse(
+  Buffer.from(gcpCredentialBase64, 'base64').toString().replace(/\n/g,"")
+)
+auth.fromJSON(credential)
+const client = new CloudTasksClient({
+  auth: auth
+})
+
+// Based on https://cloud.google.com/tasks/docs/creating-http-target-tasks#advanced_task_creation_createtask_method
+async function createHttpTask(nickname: string = 'DEFAULT') {
+  const parent = client.queuePath(project, location, queue)
+
+  // TODO: Modify this function to accept input for the following fields (as accepted by the current email template)
```

Comment:
As a next step, once this and https://github.com/johanehr/parking-reminder-frontend/pull/32 are merged, we can tie them together. As such, this TODO is left until a later PR.